src/ekore/anomalous_dimensions/unpolarized/time_like/__init__.py

```diff
@@ -1,21 +0,0 @@
-r"""The unpolarized, time-like Altarelli-Parisi splitting kernels.
```

Comment:
this file you need for sure since it is the entry point to your implementation - so instead of deleting it you should adjust it (take a look to the space-like counterpart)

Reply:
yes indeed, I had only removed it so I could add the actual functions once I am done with the as2 and as3 modules.

Reply:
but you don't need to delete the file because of that, no? and for sure we start with LO first and once we established that we step forward ...

Reply:
the file is still there, I had removed the functions which stated it is not implemented, to make way for the actual functions implementing the splitting functions. I will add the functions asap.